pinecone-mixin/config.libsonnet

```diff
@@ -0,0 +1,31 @@
+{
+  _config+:: {
+    local this = self,
+    dashboardTags: ['pinecone'],
+    dashboardPeriod: 'now-1h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '1m',
+    dashboardNamePrefix: '',
+    uid: 'pinecone',
+    
+    // Filtering and labels
+    filteringSelector: 'job=~"prometheus.scrape.pinecone_metrics"', 
+    groupLabels: [],  // Options: ['cloud'], ['region'], ['cloud', 'region'], or [] for single cloud/region
```

Comment:
Add a sensible default here. Are `cloud` and `region` not present if it's a single cloud/region?
If they're present we should use them by default